ocaml/xapi/xapi_globs.ml

```diff
@@ -1017,6 +1017,8 @@ let winbind_cache_time = ref 60
 
 let winbind_machine_pwd_timeout = ref (2. *. 7. *. 24. *. 3600.)
 
+let winbind_dns_sync_interval = ref 3600.
```

Comment:
This reminds me I really need to test my changes to replace the scheduler to actually use Mtime.Span instead of floats, and open e PR